### PR TITLE
8507: After a B/G Deployment is considered healthy, delete the old DDB and ES Cluster after a successful deploy

### PR DESCRIPTION
**Pull request by rosiehsmith**
After switch colors:
- If deploying the `prod` environment, backup the source's DynamoDB table.
- Delete the source's API gateway mappings, both non-public and public.
- Destroy the migration and migration-cron infrastructure.
- Delete the source's ElasticSearch domain.
- Delete the source's DynamoDB table in `east` and `west` regions.
- Toggle the deploy table's values for `migrate` and `source-table-version`.

Here's a successful run of this automation on [experimental1](https://app.circleci.com/pipelines/github/flexion/ef-cms/33705/workflows/e8b7eb42-61c1-4984-bbc5-6e2dd4a3af15)

Additionally: 
- updated blue-green-migration documentation
- moved `dynamo` and `elasticsearch` scripts into named folders within `./scripts`

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -791,13 +868,27 @@ workflows:
                 - irs
                 - staging
                 - test
-                - master
                 - experimental1
                 - experimental2
                 - experimental3
                 - experimental4
                 - experimental5
-                - dawson
```
Comment (mrinsin) [inline .circleci/config.yml:800]: Were these unused branches that y'all cleaned up?

Comment (rosiehsmith) [reply .circleci/config.yml:800]: Yes!

Comment (mrinsin) [reply .circleci/config.yml:800]: Awesome, thanks!

File: scripts/delete-api-gateway-mappings.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Deletes the specified api gateway mappings from the environment, for public and non-public.
+# Note: 
+# Since this is run after switch-colors, we delete the color that is not current (deploying)
```
Comment (mrinsin) [inline scripts/delete-api-gateway-mappings.sh:5]: this is me rn

**Review by mrinsin — APPROVED** [review]
This is awesome stuff!!

File: scripts/delete-api-gateway-mappings.sh (hunk 2 above)
Comment (rosiehsmith) [reply scripts/delete-api-gateway-mappings.sh:5]: blah

**Review by jimlerza — APPROVED** [review]

**Pull request merged**